uaclient/api/api.py

```diff
@@ -0,0 +1,102 @@
+from importlib import import_module
+from typing import Callable, List, Optional
+
+from uaclient.api.data_types import APIData, APIResponse, ErrorWarningObject
+from uaclient.api.errors import APIError, error_out
+from uaclient.data_types import DataObject, IncorrectFieldTypeError
+
+VALID_ENDPOINTS = ["u.pro.version.v1"]
+
+
+def call_api(endpoint_path: str, options: List[str]) -> APIResponse:
+    if endpoint_path not in VALID_ENDPOINTS:
+        return error_out(
+            APIError("{} is not a valid endpoint".format(endpoint_path))
+        )
```

Comment:
@renanrodrigo I think all the errors in this function should have specific error codes. Everything else looks great!